### ISSUE TITLE
Adding Air and godotenv to backend for development

### ISSUE THREAD

**Issue by unamdev0** [plutov/formulosity#26]
Addin 'air' and 'godotenv' packages to the Go backend to enhance development efficiency

**Comment by unamdev0** [comment]
Hi @plutov, let me know if I can implement these packages for backend. Air will help in saving time during development as developer won't have to reload the backend manually everytime to see the new changes made by them

With the help of godotenv, developer can change env settings easily instead of setting variables everytime in system

**Comment by plutov** [comment]
yes we can do that, feel free to submit a PR. The documentation needs to be updated as well probably

**Comment by CODERPP10** [comment]
Hi @plutov would like to work on updating the documentation. Please assign it to me

**Comment by unamdev0** [comment]
@CODERPP10  this has been done 
https://github.com/plutov/formulosity/pull/36
